### PR TITLE
Improve readme

### LINKED ISSUE #23: Update readme
- remove deprecated methods and add new ones

### PR DESCRIPTION
**Pull request by BeroBurny**
improve readme file

Closes #23

### CODE REVIEW

**Review by mpetrunic — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -45,11 +41,7 @@ main();
 
   - `metamaskVersion`: Metamask plugin version (by default it uses latest)
 
-- `dappeteer.setupMetamask(browser)`: returns a promise that resolves to an object that allows you to interact with MetaMask by using the following methods:
-
-  - `metamask.createAccount([password])`: it commands MetaMask to create a new account, it resolves when it's done. It can only be used while you haven't signed in yet, otherwise it throws. The password is optional, it defaults to `password1234`.
-
-  - `metamask.importAccount(seed[, password])`: it commands MetaMask to import an account, you need to provide the twelve words seed. It can only be used while you haven't signed in yet, otherwise it throws. The password is optional, it defaults to `password1234`.
```
Comment (mpetrunic) [inline README.md:52]: there is importPk method

**Review by mpetrunic — COMMENTED** [review]
@BeroBurny

File: README.md (hunk 1 above)
Comment (BeroBurny) [reply README.md:52]: yup and it works :man_shrugging:  probably create account is missing

Comment (mpetrunic) [reply README.md:52]: My comment was to add that method to readme instead of just removing. Accidentally merged 😅

**Pull request merged**